shared/src/commonMain/kotlin/io/customer/shared/tracking/api/model/Device.kt

```diff
@@ -0,0 +1,16 @@
+package io.customer.shared.tracking.api.model
+
+import io.customer.shared.common.CustomAttributes
+import io.customer.shared.getPlatform
+import kotlinx.serialization.SerialName
+
+/**
+ * Device model to be used when making device tracking request.
+ */
+@kotlinx.serialization.Serializable
+data class Device(
+    @SerialName("token") val token: String,
+    @SerialName("lastUsed") val lastUsed: Long? = null,
+    @SerialName("attributes") val attributes: CustomAttributes? = null,
+    @SerialName("platform") val platform: String? = getPlatform().name,
```

Comment:
Do we need to include `@SerialName("token")`? Can kotlinx serialization library instead use the property name? 

```kt
val token: String
    ^^^^^
```

Reply:
I think it is a good practice to add name separately
- Gives clarity and consistency in code
- Helps refactoring or renaming serialization name without affecting irrelevant code file
- I think it can help reduce challenges in code obfuscations as well